operator/scripts/prepare-olm-test.sh

```diff
@@ -8,25 +8,27 @@ DOCKER_REGISTRY="$1"
 VERSION="$2"
 PREV_VERSION="$3"
 
+UUID="$4"
```

Comment:
Maybe we could make the param optional for better UX when running locally?

Reply:
done